app/services/export_activity_to_csv.rb

```diff
@@ -127,11 +28,76 @@ def next_four_quarter_forecasts
     report_presenter.next_four_financial_quarters.map { |quarter, year| "Q#{quarter} #{year}" }
   end
 
+  private def columns
```

Comment:
Great idea to convert this into a hash to ensure we keep the order for headers and values!